src/cookies.rs

```diff
@@ -0,0 +1,123 @@
+//! Cookie-related functionality for WebDriver.
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+use webdriver::command::WebDriverCommand;
+
+use crate::client::Client;
+use crate::error;
+
+/// Type alias for a [cookie::Cookie]
+pub type Cookie<'a> = cookie::Cookie<'a>;
```

Comment:
Out of interest — why did you want to explicitly re-export the `Cookie` type here? Does that make a particular use-case easier?

Reply:
Ah, I was thinking ahead a bit for the `add_cookie()` API!

My idea was to expose the `cookie::Cookie` that `fantoccini` pulls in so that users can construct one without pulling in the crate themselves.  In addition, this would help reduce the chance of users trying to use a more recent/non-compatible version of the `cookie` crate by mistake.

But I could also be misunderstanding how Cargo handles multiple versions of a dependency!

Reply:
No, that seems totally reasonable! Good call :+1: